specification/security/resource-manager/Microsoft.Security/preview/2022-07-01-preview/applications.json

```diff
@@ -0,0 +1,537 @@
+          },
+          {
+            "$ref": "../../../common/v1/types.json#/parameters/SubscriptionId"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/ApplicationsList"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed",
+            "schema": {
+              "$ref": "../../../common/v1/types.json#/definitions/CloudError"
+            }
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink"
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/providers/Microsoft.Security/applications/{applicationId}": {
+      "get": {
+        "x-ms-examples": {
+          "Get security application by specific applicationId": {
+            "$ref": "./examples/Applications/GetApplication_example.json"
+          }
+        },
+        "tags": [
+          "Applications"
+        ],
+        "description": "Get a specific application for the requested scope by applicationId",
+        "operationId": "Application_Get",
+        "parameters": [
+          {
+            "$ref": "../../../common/v1/types.json#/parameters/ApiVersion"
+          },
+          {
+            "$ref": "../../../common/v1/types.json#/parameters/SubscriptionId"
+          },
+          {
+            "$ref": "#/parameters/ApplicationId"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/Application"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed",
+            "schema": {
+              "$ref": "../../../common/v1/types.json#/definitions/CloudError"
+            }
+          }
+        }
+      },
+      "put": {
+        "x-ms-examples": {
+          "Create application": {
+            "$ref": "./examples/Applications/PutApplication_example.json"
+          }
+        },
+        "tags": [
+          "Applications"
+        ],
+        "description": "Creates or update a security application on the given subscription.",
+        "operationId": "Application_CreateOrUpdate",
+        "parameters": [
+          {
+            "$ref": "../../../common/v1/types.json#/parameters/ApiVersion"
+          },
+          {
+            "$ref": "../../../common/v1/types.json#/parameters/SubscriptionId"
+          },
+          {
+            "$ref": "#/parameters/ApplicationId"
+          },
+          {
+            "$ref": "#/parameters/ApplicationBody"
+          }
+        ],
+        "responses": {
+          "201": {
+            "description": "Created",
+            "schema": {
+              "$ref": "#/definitions/Application"
+            }
+          },
+          "200": {
+            "description": "OK - Updated",
```

Comment:
>"description": "OK - Updated", [](http://example.com/codeflow?start=12&length=30)

When put is called on existing resource, what will happen? The entire application will be replaced and overwrite?

Reply:
yes